setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = lomap2
+description = Lead Optimization Mapper 2
+long_description = file: README.md
+long_description_type = text/markdown
+author = Gaetano Calabro and David Mobley
+author_email = 
+maintainer = Open Free Energy
+maintainer_email = openfreeenergy@omsf.io
+license = MIT
+license_file = LICENSE.txt
+url = https://github.com/OpenFreeEnergy/Lomap
+classifiers =
+    Development Status :: 3 - Alpha
```

Comment:
Shall we move it up to at least beta? Seems in moderately wide use now....

Reply:
it's 2.x, it's stable